WoWsShipBuilder.Common/Features/ShipComparison/ShipComparison.razor

```diff
@@ -274,6 +282,16 @@
 
     private MudDataGrid<GridDataWrapper> dataGrid = default!;
 
+    private int pageSize;
```

Comment:
Is it really necessary to keep pageSize and currentPage as variables? Can we not just use the properties from the grid state?